certbot/certbot/display/ops.py

```diff
@@ -106,16 +106,12 @@ def choose_values(values: List[str], question: Optional[str] = None) -> List[str
     return []
 
 
-def choose_names(installer: Optional[interfaces.Installer],
-                 question: Optional[str] = None) -> List[str]:
+def choose_names(installer: Optional[interfaces.Installer]) -> List[str]:
```

Comment:
This is a public API, usable by third party plugins (as unlikely as that is). I think usually we want to avoid breaking API compatibility.

Reply:
ah i was curious about that, but wasn't sure what counted as public api, and didn't see `choose_names` mentioned in our docs on plugin interfaces (https://eff-certbot.readthedocs.io/en/stable/contributing.html#plugin-architecture)

Reply:
I've been working under the assumption that everything outside `_internal/` is a public Python API. https://eff-certbot.readthedocs.io/en/stable/compatibility.html

Reply:
@alexzorin removed this commit, should be good to go now!